src/reference/asciidoc/testing.adoc

```diff
@@ -150,6 +150,17 @@ public class KafkaStreamsTests {
 }
 ----
 
+'''
+
+The `brokerProperties` attribute of `@EmbeddedKafka` supports standard Spring property placeholder resolution from the `Environment`.  For example:
+[source, java]
+----
+@TestPropertySource(locations = { "classpath:/kafka-broker.properties" })
+@EmbeddedKafka(topics = { "any-topic" },
+		brokerProperties = { "log.dir=${kafka.broker.logs-dir}", "listeners=PLAINTEXT://localhost:${kafka.broker.port}", "auto.create.topics.enable=true" })
+----
+In this example, the property placeholders `${kafka.broker.logs-dir}` and `${kafka.broker.port}` are resolved from the Spring `Environment`.   The `Environment` contains system properties, the system environment, _and_ properties from the classpath resource `/kafka-broker.properties`.  All of these property sources are consulted when resolving property placeholders.  If the property placeholder is resolved by the `Environment`, the resolved value will be supplied to the broker, otherwise the original value will be supplied.
```

Comment:
Would be better to start each sentence from new line: http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line.
And I don't see reason in the last one: that is really obvious that resolved values will go to the embedded broker. That is definitely the point of this `brokerProperties` option.